tutorials/plot_modifying_data_inplace.py

```diff
@@ -46,9 +47,10 @@
 # In addition, there are functions for applying the Hilbert transform, which is
 # useful to calculate phase / amplitude of your signal
 
-# Filter signal, then take hilbert transform
+# Filter signal with a fairly steep filter, then take hilbert transform
 raw_band = raw.copy()
-raw_band.filter(12, 18)
+raw_band.filter(12, 18, l_trans_bandwidth=2., h_trans_bandwidth=2.,
+                filter_length='auto', phase='zero')
 raw_hilb = raw_band.copy()
 hilb_picks = mne.pick_types(raw_band.info, meg=False, eeg=True)
 raw_hilb.apply_hilbert(hilb_picks)
```

Comment:
Maybe while you're at it: new line 60/old line 58 is missing a noun (probably 'function').